tests/mikrotik_routeros/routing_ospf_interface_print_terse/routing_ospf_interface_print_terse01.yml

```
@@ -0,0 +1,14 @@
+---
+parsed_sample:
+  - cost: "1400"
+    dynamic: ""
+    interface: "vlan11@vlan975@sfp-sfpplus1"
+    passive: ""
+  - cost: "10"
+    dynamic: ""
+    interface: "vlan10@vlan975@sfp-sfpplus1"
+    passive: ""
+  - cost: "1"
+    dynamic: ""
```

Goes along with the above template suggestion/question about capturing D for dynamic.
Thank you!

```suggestion
    dynamic: "D"
    interface: "vlan11@vlan975@sfp-sfpplus1"
    passive: ""
  - cost: "10"
    dynamic: "D"
    interface: "vlan10@vlan975@sfp-sfpplus1"
    passive: ""
  - cost: "1"
    dynamic: "D"
```